tests/proxyname_test.go

```diff
@@ -78,55 +82,70 @@ func TestProxyNameLabel(t *testing.T) { // nolint
 			// label the disabled service
 			mustOrFatal(disabledService.SetLabel(labelKey, labelValue), t)
 			return ctx
-		}).
-		Teardown(func(context.Context, *testing.T, *envconf.Config) context.Context {
-			services := []*kubernetes.Service{disabledService.(*kubernetes.Service), toggledService.(*kubernetes.Service)}
-			tools.ResetTestBoard(t, services, model)
-			return ctx
-		}).
-		Assess("should implement service.kubernetes.io/service-proxy-name", func(_ context.Context, t *testing.T, cfg *envconf.Config) context.Context {
-			ma.Logger.Info("verify the toggledServices are up.")
+		}
+	}
+
+	teardown := func(context.Context, *testing.T, *envconf.Config) context.Context {
+		for _, service := range []kubernetes.ServiceBase{disabledService, toggledService} {
+			if err := service.Delete(); err != nil {
+				t.Fatal(err)
+			}
+		}
+		return ctx
+	}
+
+	getAssessFunc := func(labelKey string) features.Func {
+		return func(_ context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			ma.Logger.Info("verify the toggledService is up without", zap.String("label", labelKey))
```

Comment:
wondering a better way to replace this `ma.Logger` for something like `log.Info`

Reply:
yeah just saw the variable `logger` in `main_test.go`. will use `logger` instead of `ma.Logger` to be cleaner.